CHANGELOG.md

```diff
@@ -21,7 +24,7 @@ and this project adheres to Rust's notion of
   generation is specified (https://github.com/zcash/zips/pull/883).
 
 ### Changed
-- Migrated to `incrementalmerkletree 0.6`, `bridgetree 0.5`.
```

Comment:
`bridgetree` is not a public dependency; it's used only for testing, so updates to the dependency version need not be reported in the CHANGELOG.